invenio_records_lom/utils/metadata.py

```diff
@@ -270,7 +299,33 @@ def set_datetime(self, datetime: str) -> None:
 
     ###############
     #
-    # methods for manipulating LOM's `technical` category
+    # methods for manipulating LOM's `metametadata` (3) category
+    #
+    ###############
+
+    def append_metametadata_contribute(
+        self,
+        name: str,
+        url: str,
+        logo: str,
+        role: str,
+    ) -> None:
+        """Append metametadata contribute."""
+        contribute = {
+            "role": {
+                "source": langstringify("LOMv1.0", lang="x-none"),
+                "value": langstringify(role, lang="x-none"),
+            },
+            "entity": name,
```

Comment:
for consistency,
since `metadata.lifecycle.contribute.<int>.entity` is of type `list[str]`,
can we make `metadata.metametadata.contribute.<int>.entity` also of type `list[str]`?

(like so, `"entity": [name],` instead of `"entity": name,`)